src/executors/default.yml

```diff
@@ -1,11 +1,11 @@
-description: >
-  This is a sample executor using Docker and Node.
-docker:
-  - image: 'cimg/node:<<parameters.tag>>'
+description: |
+  Default image to be used with this orb.
```

Comment:
If there is only a single job, might it make more sense to not include an executor? The image could instead be parameterized at the job level which seems to be easier for most folks to understand.

Unless a special execution environment is required in multiple jobs, I have been trying to lean away from creating executors.

Reply:
This is mostly a convenience thing - if we don't include an executor, we have to direct users to define their own executor, since (iirc) standard `docker` or `machine` stanzas cannot be passed through, and need to be created inside an `executor` stanza and then passed.

Reply:
In this instance, the default executor is the cimg/base. 

Within the job, could we not include an "image" parameter with the cimg/base image at the default? I recently did the same here: https://github.com/CircleCI-Public/aws-elastic-beanstalk-orb/blob/becac24396d02c4860aefa062dba0505479a1c5a/src/jobs/deploy.yml#L13

Reply:
Sure, will opt for that approach.